smac/intensification/intensification.py

```diff
@@ -237,192 +558,283 @@ def eval_challenger(self,
                 self.logger.info("Maximum #runs for intensification reached")
                 self._next_iteration()
 
-            if not self.use_ta_time_bound and self.elapsed_time - time_bound >= 0:
-                self.logger.info("Wallclock time limit for intensification reached (used: %f sec, available: %f sec)",
-                                 self.elapsed_time, time_bound)
+            if not self.use_ta_time_bound and elapsed_time - time_bound >= 0:
+                self.logger.info(
+                    "Wallclock time limit for intensification reached "
+                    "(used: %f sec, available: %f sec)",
+                    elapsed_time,
+                    time_bound
+                )
+
                 self._next_iteration()
 
             elif self._ta_time - time_bound >= 0:
-                self.logger.info("TA time limit for intensification reached (used: %f sec, available: %f sec)",
-                                 self._ta_time, time_bound)
+                self.logger.info(
+                    "TA time limit for intensification reached (used: %f sec, available: %f sec)",
+                    self._ta_time, time_bound
+                )
+
                 self._next_iteration()
 
         inc_perf = run_history.get_cost(incumbent)
 
         return incumbent, inc_perf
 
-    def _add_inc_run(self,
-                     incumbent: Configuration,
-                     run_history: RunHistory,
-                     log_traj: bool = True) -> None:
-        """Add new run for incumbent
+    def _get_next_inc_config(self,
```

Comment:
Could you rename this to `_get_next_inc_run` as it does not return a configuration?

Reply:
understood, I will rename it.